src/widget/wcollapsiblegroupbox.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <QGroupBox>
+#include <QProxyStyle>
+
+/// Custom style for QGroupBox that renders the 'enabled' checkbox as ArrowDown/ArrowRight
+/// icon and skips drawing the groupbox frame when the box is collapsed.
+class CollapsibleGroupBoxStyle : public QProxyStyle {
+    Q_OBJECT
+  public:
+    explicit CollapsibleGroupBoxStyle(QStyle* pStyle = 0)
```

Comment:
Should this be `nullptr` instead of `0`?

Reply:
Umm, right.
Seems I mixed up the original Qt implementation with that of [ctkCollapsibleGroupBox](https://github.com/commontk/CTK/blob/master/Libs/Widgets/ctkCollapsibleGroupBox.cpp)

Reply:
```suggestion
    explicit CollapsibleGroupBoxStyle(QStyle* pStyle = nullptr)
```